letter_opener_web.gemspec

```
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'actionmailer', '>= 6.1'
   gem.add_dependency 'letter_opener', '~> 1.9'
   gem.add_dependency 'railties', '>= 6.1'
-  gem.add_dependency 'rexml'
+  gem.add_dependency 'rexml', '~> 3.3.2'
```

This will lock us to the `3.3.x` line, meaning if/when `3.4.x` is released, this constraint won't allow folks to upgrade to it. So I think something like this might be better:

```suggestion
  gem.add_dependency 'rexml', '>= 3.3.2'
```

Though, I also wonder if this is more of a thing that users of this gem should worry about, since they might have specific version needs/constraints for `rexml`.

🤔💭